apps/crn-server/src/validation/research-output.validation.ts

```diff
@@ -85,3 +100,50 @@ export const validateResearchOutputPostRequestParameters = validateInput(
     coerce: false,
   },
 );
+
+export const validateResearchOutputPostRequestParametersIdentifiers = (
```

Comment:
im sure you have looked into it but doesn't ajv support something similar? we are saying that at least one and only one of the identifiers is required for when asapFunded and usedInPublication are present, correct? 

I guess it's a little more complex with the other condition since we have a list of types that support each of these indetifiers, correct?

Reply:
Yes ajv theoretically supports it. But the setup is so complex, with nested levels of field attributes that I decided this approach is easier to test and to understand.

Reply:
fair, im cool with that